handlers/aws/cloudwatch_logs_trigger.py

```diff
@@ -94,25 +96,28 @@ def _handle_cloudwatch_logs_event(event: dict[str, Any], aws_region: str) -> Ite
             range_start=0,
         )
 
-        for log_event, ending_offset, starting_offset, newline_length in events:
+        for log_event, json_object, ending_offset, starting_offset, newline_length in events:
             assert isinstance(log_event, bytes)
 
-            es_event = deepcopy(_default_event)
-            es_event["@timestamp"] = datetime.datetime.now().strftime("%Y-%m-%dT%H:%M:%S.%fZ")
-            es_event["fields"]["message"] = log_event.decode("UTF-8")
+            for extracted_log_event, extracted_starting_offset, is_last_event_extracted in extract_events_from_field(
+                log_event, json_object, starting_offset, ending_offset, integration_scope, extractor_events_from_field
+            ):
+                es_event = deepcopy(_default_event)
+                es_event["@timestamp"] = datetime.datetime.now().strftime("%Y-%m-%dT%H:%M:%S.%fZ")
```

Comment:
I am not 100% sure about this, it's just a double check.

The use of the `Z` in the  `%Y-%m-%dT%H:%M:%S.%fZ` format implies this is a UTC timezone, right?

The use of `datetime.datetime.now()` to get a `datetime` object might return a local date and time.

Reply:
good catch!

it is not only here, btw, but on every input handler

I've chosen for format mimicking filebeat, forgetting about the timezone implications

we should use `datetime.datetime.utcnow()`

Reply:
I was bitten from Python's timezone support in the past ^^